tests/test_global_settings_handlers.cpp

```diff
@@ -267,6 +267,32 @@ TEST_F(TestGlobalSettingsHandlers, daemonRegistersHandlerThatAcceptsValidBackend
     ASSERT_NO_THROW(handler->set(key, val));
 }
 
+TEST_F(TestGlobalSettingsHandlers, daemonRegistersHandlerThatTransformsHyperVDriver)
+{
+    auto key = mp::driver_key, val = "hyper-v";

```

Comment:
Two minor things here: 

First one is the multiple variables declaration in one line. I know we have other code doing this and the code here is correct, but this is not a good practice in general in terms of readability and error proneness. There is also a c++ core guideline [item](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#es10-declare-one-name-only-per-declaration) for this. 

Second one is that `const` can be added to the three to make these variables read only. That makes the result code: 
```
    const auto key = mp::driver_key;
    const auto val = "hyper-v";
    const auto transformed_val = "hyperv";
```